packages/playwright-test/types/test.d.ts

```diff
@@ -730,8 +731,33 @@ interface TestConfig {
    * });
    * ```
    *
+   * You can also use multiple servers:
+   *
+   * ```ts
+   * // playwright.config.ts
+   * import { PlaywrightTestConfig } from '@playwright/test';
+   * const config: PlaywrightTestConfig = {
+   *   webServer: [
```

Comment:
i wish it'd be called 'services'! Like [Github Actions services](https://docs.github.com/en/actions/using-containerized-services/about-service-containers)